content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -15,7 +15,7 @@ weight: 70
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
```

Comment:
BTW it's actually OK to leave this line unchanged.

Reply:
In the text of the page, signpost readers to read the version of this page for _their_ version of Kubernetes - so that they are viewing accurate documentation for this alpha feature.

Reply:
This is an unusual case as this enhancement has been reimplemented in Kubernetes 1.20 and any docs prior to 1.20 (1.16 though 1.19) are incorrect. Given that, how do you suggest we signal that these docs only are relevant to this version of this alpha enhancement and not the prior implementation?

Reply:
My suggestion for this is / was: https://github.com/kubernetes/website/pull/24725#discussion_r516152624

If people are using v1.18 and read the v1.18 docs, the two should match.